06-files-input-and-output/high_scores.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+import csv
+import os
+
+def parse_csv(csv_file):
+    data = {}
+    with open(csv_file, "r") as csv_file:
+        try:
+            csv_reader = csv.reader(csv_file)
+            for name, score in csv_reader:
+                score = int(score)
+                data[name] = max(data.get(name) or 0, score)
+        except ValueError as e:
```

Comment:
I don't agree that `except ValueError` catches csv.Error as well. You might need to catch 2 kinds of errors regardless but one doesn't catch the other. Do you understand why?
Please handle `csv.Error` as well.